bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
I think it would be good to make such an announcement.  The error message is somewhat unclear (Why does pinning break my code?  What does it mean that dev doesn't match < 4.0.0?) and the fix is painful.  I have a package waiting to be released which I removed all of the upper version bounds from because it was too much of a pain to deal with all of the conflicts when pins are involved.

Should these `version: dev` lines be actively removed?  In my experience so far they cause more harm than good.  I can start something on discuss.ocaml.org if you're interested in chiming in.